apis/apiextensions/fn/proto/v1beta1/run_function.pb.go

```diff
@@ -15,7 +15,7 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.33.0
+// 	protoc-gen-go v1.25.0-devel
```

Comment:
I wouldn't expect this file to be changing at all, especially for it to be reverting this back to an older version of `protoc-gen-go`.

Any idea why this change is being included? 🤔

Reply:
@jbw976 Yea, it happened because my protoc-gen-go version was lower so it modified that latest version in `run_function.pb.go`. I have fixed that issue now